cli/command/system/version.go

```diff
@@ -17,7 +16,7 @@ import (
 
 var versionTemplate = `Client:
  Version:      {{.Client.Version}}
- API version:  {{.Client.APIVersion}}
+ API version:  {{.Client.ActualAPIVersion}} (default version {{.Client.DefaultAPIVersion}})
```

Comment:
What is the difference with old way?  Or I don't really understand "docker version output is not consistent".